src/components/operators.jl

```diff
@@ -445,7 +445,8 @@ function parse_operator(ps::ParseState, ret::EXPR, op::EXPR)
     elseif is_prime(op)
         if isidentifier(ret) || isliteral(ret) ||
             headof(ret) in (:call, :tuple, :brackets, :ref, :vect, :vcat, :hcat, :typed_vcat, :typed_hcat, :comprehension, :typed_comprehension, :curly, :braces, :braces_cat) ||
-            headof(ret) === :do
+            headof(ret) === :do ||
+            is_dot(headof(ret))
```

Comment:
@ZacLN Why do we have these checks anyways? Isn't `'` valid basically everywhere?

Reply:
Not for blocks (e.g. `begin end'`), it maybe better to move to an exclusionary list